README.md

```diff
@@ -1,175 +1,22 @@
-```
-
-We can now directly execute our `Hello World` application on a real connected device or an emulator:
-```console
-$ cargo apk run
-```
-
-If the crate includes a runnable binary as suggested above, you will likely be greeted by the following error:
-
-```console
-$ cargo apk run
-error: extra arguments to `rustc` can only be passed to one target, consider filtering
-the package by passing, e.g., `--lib` or `--bin NAME` to specify a single target
-Error: Command `cargo rustc --target aarch64-linux-android -- -L hello_world_android/target/cargo-apk-temp-extra-link-libraries` had a non-zero exit code.
-```
-
-To solve this, add `--lib` to the run invocation, like so:
-```console
-$ cargo apk run --lib
-```
-
-### 6. Inspect the output
-
-`ndk-glue` redirects stdout and stderr to Android `logcat`, including the `println!("Hello World")` by the example above. See [Logging and stdout](##Logging-and-stdout) below how to access it.
-
-## Rendering to the window
-
-Android native apps have no easy access to [Android's User Interface](https://developer.android.com/guide/topics/ui) functionality (bar [JNI](##jni) interop). Applications can instead draw pixels directly to the window using [`ANativeWindow_lock`](https://developer.android.com/ndk/reference/group/a-native-window#group___a_native_window_1ga0b0e3b7d442dee83e1a1b42e5b0caee6), or use a graphics API like OpenGL or Vulkan for high performance rendering.
-
-## Logging and stdout
-Stdout is redirected to the android log api when using `ndk-glue`. Any logger that logs to
-stdout, like `println!`, should therefore work.
-
-To filter on this output in `logcat`:
-```console
-$ adb logcat RustStdoutStderr:D *:S
-```
-
-### Android logger
-Enable the `"logger"` feature on the `ndk-glue` macro and configure its log tag and debug level through the attribute macro:
-
-`src/lib.rs`
-```rust
-#[cfg_attr(target_os = "android", ndk_glue::main(logger(level = "debug", tag = "my-tag")))]
-pub fn main() {
-    log!("hello world");
-}
-```
-
-## App/APK configuration
-Android APKs contain a file called `AndroidManifest.xml`, which has things like permission requests and feature declarations, plus configuration of activities, intents, resource folders and more. This file is autogenerated by `cargo-apk`. To control what goes in it through Cargo.toml, refer to [`cargo-apk`'s README](./cargo-apk/README.md).
-
-## Overriding crate paths
-The macro `ndk_glue::main` tries to determine crate names from current _Cargo.toml_.
-You can override this names with specific paths like so:
-```rust
-#[ndk_glue::main(
-  ndk_glue = "path::to::ndk_glue",
-)]
-fn main() {}
-```
 
-## JNI
-Java Native Interface (JNI) allows executing Java code in a VM from native applications. To access
-the JNI use the `AndroidContext` from the `ndk-context` crate. `ndk-examples` contains a `jni_audio`
-example which will print out all output audio devices in the log.
 
-- [`jni`](https://crates.io/crates/jni), JNI bindings for Rust
```

Comment:
Since there's _some_ tie-in between JNI and NDK it might be useful to retain minimal documentation around this? Not a hard requirement, probably in-line with your idea/comment to spend some more time later to write concise guides how to use this (these repos) in conjunction with each other?

Reply:
Let's tackle this at a later point.